fullstop-plugin-api/src/main/java/org/zalando/stups/fullstop/events/CloudtrailEventSupport.java

```diff
@@ -100,6 +97,10 @@ public static String getEventId(final CloudTrailEvent event) {
      */
     public static String getAccountId(final CloudTrailEvent event) {
         CloudTrailEventData eventData = getEventData(event);
+        // otherwise we get a java.lang.ClassCastException in tests
+        if (eventData instanceof TestCloudTrailEventData) {
```

Comment:
Production code should not have any test-dependencies. Is it possible to provide a better mock in those tests, to avoid ClassCastExceptions?